renderer/components/editor/index.js

```diff
@@ -66,6 +66,7 @@ export default class Editor extends React.Component {
             justify-content: center;
             font-size: 1.4rem;
             color: #fff;
+            margin-left: -72px;
```

Comment:
This is fine for now, setting it to `absolute` is definitively an option, but we can look into that when we make changes to the traffic lights or titlebar in the future. Is this consistent with how we're doing this in other windows that have custom traffic lights and titlebars? Thank you!

Reply:
That makes sense, thank you @joviij. I was just double checking, we should *only* be using the custom traffic lights for this window.